deployments/Dockerfiles/tendermint/tendermint.py

```diff
@@ -111,6 +118,7 @@ def build_node_command(self, debug: bool = False) -> List[str]:
             f"--p2p.laddr={self.p2p_laddr}",
             f"--p2p.seeds={p2p_seeds}",
             f"--consensus.create_empty_blocks={str(self.consensus_create_empty_blocks).lower()}",
+            f"--abci={'grpc' if self.use_grpc else 'socket'}",
```

Comment:
should be used on `tendermint node`. We make it explicit, also when using the default: TCP (socket)